src/Compilers/CSharp/Portable/Emitter/NoPia/EmbeddedType.cs

```diff
@@ -56,7 +56,7 @@ public void EmbedAllMembersOfImplementedInterface(SyntaxNode syntaxNodeOpt, Diag
         protected override int GetAssemblyRefIndex()
         {
             ImmutableArray<AssemblySymbol> refs = TypeManager.ModuleBeingBuilt.SourceModule.GetReferencedAssemblySymbols();
-            return refs.IndexOf(UnderlyingNamedType.ContainingAssembly, ReferenceEqualityComparer.Instance);
+            return refs.IndexOf(UnderlyingNamedType.ContainingAssembly, 0, ReferenceEqualityComparer.Instance);
```

Comment:
>IndexOf [](start = 24, length = 7)

Which `IndexOf` operator is this?  Can you please use a named argument for `0`?

Reply:
Sure. It was actually previously using `IImmutableList.IndexOf()` extension because `ImmutableArray` does not have IndexOf with 2 parameters :(

Reply:
I'm gonna add extension method
```C#
internal static int IndexOf<T>(this ImmutableArray<T> array, T item, IEqualityComparer<T> comparer)
            => array.IndexOf(item, startIndex: 0, comparer);
```

to address that globally.

Reply:
OK, but I see a receiver and 3 parameters. So I'm looking for a 3-parameter instance method or a 4-parameter extension method, and I don't see one.

---
In reply to: [394065299](https://github.com/dotnet/roslyn/pull/42451#discussion_r394065299) [](ancestors = 394065299)

Reply:
> Oh, the 0 is startIndex and this is an instance method?

Correct.